src/components/ProjectCard.jsx

```diff
@@ -67,37 +79,74 @@ function ProjectCard({
 }) {
   const classes = useStyles();
 
-  return (
-    <Paper className={classes.root}>
-      <div className={classes.icon}>
-        <PetsIcon color="inherit" />
-      </div>
-      <div className={classes.projectHeader}>
-        <Typography variant="h6" className={classes.projectName}>
-          <Link to={'project/' + title.split(' ').join('-')}>{title}</Link>
+  const getProjectLink = (title) => {
+    return `/project/${title.split(' ').join('-')}`;
+  };
+
+  const projectLogo = (
+    <div className={classes.icon}>
+      <Avatar alt={title} src={logo} />
+    </div>
+  );
+
+  const projectThemes = (
+    <div className={classes.themes}>
+      {themes.map((theme, index) => (
+        <Button
+          key={theme.name}
+          variant="outlined"
+          color={index % 2 === 0 ? 'primary' : 'secondary'}
+          size="small"
+        >
+          {theme.name}
+        </Button>
+      ))}
+    </div>
+  );
+
+  const projectHeader = (
+    <div className={classes.projectHeader}>
+      <Typography variant="h6" className={classes.projectName}>
+        <Link to={getProjectLink(title)}>{title}</Link>
+      </Typography>
+      <div className={classes.projectMeta}>
+        <Typography variant="caption" component="span">
+          {member_count} contributing
+        </Typography>
+        <div className={classes.separator} />
+        <Typography variant="caption" component="span">
+          {interested_count} interested
         </Typography>
-        <div className={classes.projectDetails}>
-          <Typography variant="caption" component="span">
-            {member_count} contributing
-          </Typography>
-          <div className={classes.separator} />
-          <Typography variant="caption" component="span">
-            {interested_count} interested
-          </Typography>
-        </div>
-      </div>
-      <div className={classes.themes}>
-        {themes.map((theme, index) => (
-          <Button
-            key={theme.name}
-            variant="outlined"
-            color={index % 2 === 0 ? 'primary' : 'secondary'}
-            size="small"
-          >
-            {theme.name}
-          </Button>
-        ))}
       </div>
+    </div>
+  );
+
+  const projectInfo = (
+    <div>
+      <Typography>{oneliner}</Typography>
+    </div>
+  );
+
+  return (
+    <Paper className={classes.root}>
+      {variant === 'profile' ? (
+        <>
+          {projectLogo}
+          <div>
+            <div className={classes.container}>
+              {projectHeader}
+              {projectThemes}
+            </div>
+            {projectInfo}
+          </div>
+        </>
+      ) : (
+        <>
```

Comment:
I like the idea of the project logo being on the stub! The catch seems to be that the spacing doesn't seem to shrink if the logo is small.

![image](https://user-images.githubusercontent.com/47229249/87070912-56171b00-c1ce-11ea-9754-20dc1d28336c.png)